base/expr.jl

```diff
@@ -35,11 +46,36 @@ copy_exprargs(x::Array{Any,1}) = Any[copy_exprs(a) for a in x]
 ==(x::Expr, y::Expr) = x.head === y.head && isequal(x.args, y.args)
 ==(x::QuoteNode, y::QuoteNode) = isequal(x.value, y.value)
 
+"""
+    expand(x)
+
+Takes the expression `x` and returns an equivalent expression in lowered form.
```

Comment:
Maybe just added a cross reference to code_lowered and the corresponding dev-doc for the content and structure of lowered code?